### ISSUE TITLE
Error when loading evil-noautochdir

### ISSUE THREAD

**Issue by isovector** [gbarta/evil-noautochdir#1]
Hi there. I'm super new to emacs, so I'm probably doing something stupid here, but I get this when `require`ing evil-noautochdir.

```
Debugger entered--Lisp error: (void-variable evil-cd)
  byte-code("\303\304\305\306\307$\210\310\304\307\"\210\311\301 \312\307  ?\205\313\211\314    !\203*\315 !\316\n!\202.\317\320 \"&\210\321\322\300\"\207" [evil-cd dir evil-cwd ad-add-advice evil-ex (evil-ex-noautochdir nil t (advice lambda nil (with-evil-cwd ad-do-it))) around nil ad-activate evil-define-command "Change directory." "~" file-directory-p expand-file-name message error "Can't find directory %s in path" evil-ex-define-cmd "cd"] 9)
  require(evil-noautochdir)
  eval((require (quote evil-noautochdir)) nil)
  eval-last-sexp-1(nil)
  eval-last-sexp(nil)
  call-interactively(eval-last-sexp nil nil)
```

The relevant bits of my `~/.emacs` are:

``` elisp
(require 'package)
(push '("marmalade" . "http://marmalade-repo.org/packages/")
      package-archives )
(push '("melpa" . "http://melpa.milkbox.net/packages/")
      package-archives)
(package-initialize)

(require 'evil)
(evil-mode 1)

(add-to-list 'load-path "~/.emacs.d/el-get/el-get")
(unless (require 'el-get nil 'noerror)
  (with-current-buffer
      (url-retrieve-synchronously
       "https://raw.githubusercontent.com/dimitri/el-get/master/el-get-install.el")
    (goto-char (point-max))
    (eval-print-last-sexp)))
(add-to-list 'el-get-recipe-path "~/.emacs.d/el-get-user/recipes")

(el-get-bundle gbarta/evil-noautochdir)
(require 'evil-noautochdir)
```

Any help you might be able to shed would be fantastic. I'm running GNU Emacs 24.3.1.

Thanks!

**Comment by gbarta** [comment]
Thanks for the bug report!  I feel like I am still quite new to Emacs myself :)

It appears that the problem is because el-get is automatically byte-compiling the file, and the file is not quite right for byte-compilation to work.  I think all it requires is declaring the dependency on evil rather than just assuming it has already been loaded.

Hopefully you will find the problem has been fixed by commit 9a079c02c745fcfad845d88c7368bb9e7f02d384

**Comment by isovector** [comment]
That fixes it! Thanks for your work, and the super quick turnaround on this!
